staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -182,7 +187,7 @@ func GetResourceKind(groupVersion schema.GroupVersion, storage rest.Storage, typ
 	return fqKindToRegister, nil
 }
 
-func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storage, ws *restful.WebService) (*metav1.APIResource, error) {
+func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storage, ws *restful.WebService) (*metav1.APIResource, *storageversion.ResourceInfo, error) {
```

Comment:
I'm not really a fan of adding a return parameter, but I'm not sure what would be better. I guess it'd be weird to store things in the APIInstaller for reading later?

Reply:
another option is to group together `*metav1.APIResource` and `*storageversion.ResourceInfo` (what the [original PR](https://github.com/kubernetes/kubernetes/pull/84016/commits/c27760c5c6f6a8a5c572c6806ad349d1d73bb4f0#diff-a8da7b9bda33b6055a1f5deb7d0c33058c5d706ec5b44c665191a359a43160c6R195) did) in a bigger "API resource info". Later we need to [extract them separately](https://github.com/kubernetes/kubernetes/pull/84016/commits/c27760c5c6f6a8a5c572c6806ad349d1d73bb4f0#diff-43ce7444baa0a48d4a4ec37170c8def820afbeab7b16b5df37894d73b96d9071R111) because they are used in different places.

Reply:
I guess I won't ask for a change.